packages/dataviews/src/style.scss

```diff
@@ -209,31 +209,43 @@
 		&:last-child {
 			border-bottom: 0;
 		}
+		&:hover,
+		&:focus-within {
+			background-color: lighten($gray-100, 3%);
+		}
 	}
 
-	.dataviews-list-view__item {
-		padding: $grid-unit-15 $grid-unit-40;
+	.dataviews-list-view__item.is-selected,
+	.dataviews-list-view__item.is-selected:hover,
+	.dataviews-list-view__item.is-selected:focus-within {
+		background-color: $gray-100;
+
+	}
+
+	.dataviews-list-view__item-content {
+		padding: $grid-unit-15 0 $grid-unit-15 $grid-unit-40;
+		width: 100%;
 		cursor: default;
-		&:focus,
```

Comment:
The `background-color` is now controlled by the parent. The `focus` is handled via the `::after` pseudo-element.